internal_ws/ykshim/src/prod_api.rs

```diff
@@ -1,3 +1,5 @@
+//! The ykshim API.
```

Comment:
Can we expand this? It won't be very obvious to a new reader what `ykshim` is or what `prod` means (*I'm* only vaguely sure!).

Reply:
`prod` is a term that will disappear. We agreed to merge `prod.rs` into `lib.rs`.

(Note there's already a comment in `lib.rs` which [explains what this is all about](https://github.com/softdevteam/yk/blob/8be961bb1e797253dd40a87a047de56cbd5c75d0/internal_ws/ykshim/src/lib.rs#L1-L4).)

Reply:
(I'll do this change as my next PR, so that it doesn't get forgotten)

Reply:
Excellent!